VAMobile/src/utils/formattingUtils.test.ts

```diff
@@ -0,0 +1,23 @@
+import { getFormattedTimeForTimeZone } from 'utils/formattingUtils'
+
+describe('formatting utils: getFormattedTimeForTimeZone', () => {
```

Comment:
These unit tests are exhaustive so I figured that we don't need to add any detox tests. Let me know if I should still add one though.